### ISSUE TITLE
Copy/Paste (CTRL+C/CTRL+V) on the tree does not match user expectation

### ISSUE THREAD

**Issue by MWEclipse** [MWEclipse/testing#184]
| --- | --- |
| Bugzilla Link | [542790](https://bugs.eclipse.org/bugs/show_bug.cgi?id=542790) |
| Status | NEW |
| Importance | P3 normal |
| Reported | Dec 14, 2018 02:54 EDT |
| Modified | May 23, 2023 00:36 EDT |
| Version | 1.18.0 |
| Reporter | Tamas Miklossy |

## Description 
Currently, A copied element can be pasted into the tree view only once. The user has to release both the CTRL and the V keys to be able to paste the copied element once more. Reproducible on all EMFForms-based editors (e.g: EcoreEditor).

I would expect that if the user holds the CTRL key, releases the V key and presses immediately the V key again without releasing the CTRL key, the element is pasted once again.

**Comment by MWEclipse** [comment]
By Eugen Neufeld on Jan 11, 2019 09:26

Thank you for the report. \
We would happily accept a contribution for this.

**Comment by MWEclipse** [comment]
Jun 01, 2021 05:44

This bug hasn't had any activity in quite some time. Maybe the problem got resolved, was a duplicate of something else, or became less pressing for some reason - or maybe it's still relevant but just hasn't been looked at yet.

If you have further information on the current state of the bug, please add it. The information can be, for example, that the problem still occurs, that you still want the feature, that more information is needed, or that the bug is (for whatever reason) no longer relevant.

--\
The automated Eclipse Genie.

**Comment by MWEclipse** [comment]
May 23, 2023 00:36

This bug hasn't had any activity in quite some time. Maybe the problem got resolved, was a duplicate of something else, or became less pressing for some reason - or maybe it's still relevant but just hasn't been looked at yet.

If you have further information on the current state of the bug, please add it. The information can be, for example, that the problem still occurs, that you still want the feature, that more information is needed, or that the bug is (for whatever reason) no longer relevant.

--\
The automated Eclipse Genie.
